bindep.txt

```diff
@@ -0,0 +1,3 @@
+# List your controller-side system package requirements here if exist
+# to support easy execution environments building for your collection users
+# https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html
```

Comment:
I would probably move these files into `meta/` and add an EE config which references them. That avoids polluting the collection's root directory.

Reply:
I personally prefer to see them in root as it feels like important information that can be missed if it's in `meta/`. I mean it's important in general, not just for EE's purposes.
Also the config would be another additional file. Maybe collections have no controller-side dependencies. If maintainers don't notice, it can lead to more pollution with unnecessary files.
How about maybe putting a note to the files that `if you don't have any dependencies, just remove the file`?